.github/workflows/bundle-analyser.yml

```diff
@@ -16,21 +16,27 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - run: npm install --global corepack@0.31.0
+
+      - run: corepack enable
+        shell: bash
+        working-directory: support-frontend
+
       - name: Install Node
         uses: actions/setup-node@v4
         with:
           node-version-file: '.nvmrc'
       # Cache npm dependencies using https://github.com/bahmutov/npm-install
-      - name: Cache dependencies
-        uses: bahmutov/npm-install@v1
-        with:
-          working-directory: support-frontend
+#      - name: Cache dependencies
```

Comment:
Should we delete this if it's no longer needed?

Reply:
I've left these caching bits commented as I'd like to try putting them back now we've migrated. I was seeing weirdness before which I put down to having some branches on yarn while mine was on pnpm and caching across branches not working well in that case.